fhir-model/src/main/java/com/ibm/fhir/model/visitor/DefaultVisitor.java

```diff
@@ -29,8 +29,8 @@
  * <ul>
  * <li>preVisit methods to control whether a given Resource or Element gets visited
  * <li>visitStart methods to provide setup behavior prior to the visit
- * <li>supertype visit methods to perform some common action on all visited Resources and Elements
- * <li>subtype visit methods to perform unique behavior that varies by the type being visited
+ * <li>defaultAction methods to perform some common action on all visited Resources and Elements
+ * <li>specific visit methods to perform unique behavior that varies by the type being visited
```

Comment:
I forgot this class was generated and I updated this javadoc by hand.  Regenerating the model has overwritten these edits.  Mind updating the codeGenerator to preserve the `supertype`/`subtype` version?

Reply:
got it.  updating the code generator